security/crowdsec/src/opnsense/mvc/app/controllers/OPNsense/CrowdSec/Api/ServiceController.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+// SPDX-License-Identifier: MIT
+// SPDX-FileCopyrightText: © 2021 CrowdSec <info@crowdsec.net>
+
+namespace OPNsense\CrowdSec\Api;
+
+use OPNsense\Base\ApiControllerBase;
+use OPNsense\Core\Backend;
+
+/**
+ * Class ServiceController
+ * @package OPNsense\Cron
+ */
+class ServiceController extends ApiControllerBase
```

Comment:
this might be missing out on a lot of automatic glue and future fixes not using mutable controller type

Reply:
So far I had a "whatever works" approach towards the PHP code, I'll have a better look. I suppose I can change it later